test/instrumental_test.js

```diff
@@ -103,12 +68,57 @@ test('metricPrefix ending with dots wont send double dots', function (t) {
   now = Math.round(new Date().getTime() / 1000);
   dummy_events =  { on: function(e){ } };
 
-  config.instrumental.metricPrefix = "testprefix.";
-  instrumental.init(now, config, dummy_events);
+  TestHelper.config.instrumental.metricPrefix = "testprefix.";
+  TestHelper.instrumental.init(now, TestHelper.config, dummy_events);
 
-  payload = instrumental.build_payload(metrics);
+  payload = TestHelper.instrumental.build_payload(metrics);
 
   t.assert(payload.indexOf("increment testprefix.my.test.1 2805 ") > -1, "Metric name was not prefixed properly (got " + JSON.stringify(payload) + ")");
 
   t.end();
 });
+
+test("by default no messages are logged on every meetric send", function (t) {
+  oldTime = Math.round(new Date().getTime() / 1000);
+
+  // the default config value in production, though the default in test is true
+  TestHelper.config.debug = false;
+
+  var metricName = "test.metric"+Math.random();
+  sendMetric(metricName, oldTime);
+  sendMetric(metricName, oldTime, {skipInit: true});
+  sendMetric(metricName, oldTime, {skipInit: true});
+  sendMetric(metricName, oldTime, {skipInit: true});
+  sendMetric(metricName, oldTime, {skipInit: true});
+  sendMetric(metricName, oldTime, {skipInit: true});
+
+  checkForMetric(metricName, {
+    expectedSum: 6, // failures don't get sent
+    found: function(){
+      var cert_log_messages =
+        TestHelper.log.filter(function(entry){return true});
+      var expected_messages = [
+        "Adding node default ssl cert option",
```

Comment:
do we not have timestamps, or does the checker strip those?

Reply:
"No"

"That would be a good thing to add, but probably out of scope"

"I agree on both fronts"